.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: python
 sudo: false
-python: 2.7
+python: 3.7
 env:
-    - TOX_ENV=py27
-    - TOX_ENV=py34
+    - TOX_ENV=py37
+    - TOX_ENV=py36
     - TOX_ENV=flake8
 matrix:
```

Comment:
We can remove this section now. We aren't testing against 3.4, so it doesn't make sense to "allow" that nonexistent env to fail.